sdk/formrecognizer/tests.yml

```diff
@@ -5,6 +5,7 @@ stages:
     parameters:
       ServiceDirectory: formrecognizer
       timeoutInMinutes: 120 # how long to run the job before automatically cancelling
+      SupportedClouds: 'Public,Canary,UsGov,China'
```

Comment:
not sure what the default regions are in Java, but in .NET `Public` targets `westus2` and because the service has asked us to stay in `canadacentral` I had to do this: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/tests.yml#L8